.gitlab-ci.yml

```diff
@@ -11,7 +11,7 @@ variables:
 
 include:
   - project: mattermost/ci/mattermost-webapp
-    ref: release-6.0
+    ref: master
```

Comment:
intended?

Reply:
yes, both intended reverting this unintended change https://github.com/mattermost/mattermost-webapp/pull/8642#issuecomment-901799779

Reply:
Thanks for the context